src/rm_cms.c

```diff
@@ -32,6 +35,19 @@ static int GetCMSKey(RedisModuleCtx *ctx, RedisModuleString *keyName, CMSketch *
     return REDISMODULE_OK;
 }
 
+static int GetCMSKeyIgnore(RedisModuleCtx *ctx, RedisModuleString *keyName, CMSketch **cms,
```

Comment:
I believe this was added so in case a key is missing, the command can still succeed.
IMO, it should fail.
@filipecosta90 @gkorland WDYT?

Reply:
Thx for review my code.
IMO, The entire request should not be erroneous because one key does not exist in the batch command.
But it's really easy to ignore some errors.
If everyone thinks that exceptions should be thrown, I will fix it.

Reply:
Thank you. I will let you know...

Reply:
Hello, I have modified the code.
Use the [NX/XX] to let the user decide whether to ignore exceptions.
What do you think.